### PR TITLE
Output summary

### PR DESCRIPTION
**Pull request by matchastack**
Refactored code for better scalability and added output in the form `{ carpark name : tuple(...}`

### CODE REVIEW

File: des/DES.py [hunk 1]
```diff
@@ -4,76 +4,114 @@
...
+    'staff' : 0.5, 
+    'esp' : 0.05
+}
+
+def custom_choice(items, prob):
     rng = np.random.default_rng()
-    return rng.choice(list, p=prob)
+    return rng.choice(items, p=prob)
+
+def create_cp(env, cp_dict):
```
Comment (sunroofgod) [inline des/DES.py]: Nit: might be clearer to declare the typings of params and docstrings

File: des/car.py [hunk 2]
```diff
@@ -1,14 +1,26 @@
+import numpy as np
 class Car:
 
     def __init__(self, id : int, type="hourly"):
         self.id = id
         self.type = type.lower()
 
+    def name(self):
```
Comment (sunroofgod) [inline des/car.py]: Nit: name might be unintuitive since you're retrieving the id attribute instead

Comment (matchastack) [reply des/car.py]: will change to `get_id`

File: des/DES.py (hunk 1 above)
Comment (matchastack) [reply des/DES.py]: will do

**Review by sunroofgod — APPROVED** [review]
LGTM!

**Pull request merged**